### PR TITLE
Add index.rst with quickstart statement

### PR DESCRIPTION
**Pull request by shangrex**

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -6,10 +6,23 @@
 Welcome to Language Model Playground's documentation!
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
+and it can generate sentences, evaluation models, inference words. 
+Get started with :ref:`my-reference-label`.
```
Comment (ProFatXuanAll) [inline doc/source/index.rst]: You can reference section in `.rst` directly, no need to create label.

File: doc/source/index.rst [hunk 2]
```diff
@@ -6,10 +6,23 @@
 Welcome to Language Model Playground's documentation!
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
```
Comment (ProFatXuanAll) [inline doc/source/index.rst]: Put single sentence in the one line, only use multiline when passing length limit (80 characters)

File: doc/source/index.rst [hunk 3]
```diff
@@ -6,10 +6,23 @@
...
+and it can generate sentences, evaluation models, inference words.
```
Comment (ProFatXuanAll) [inline doc/source/index.rst]: ~evaluataion~ evaluate.
Remove `inference words` and add following utilites:

- Train tokenizer
- Train neural network based language model
- Generate text using language model
- And more to come

File: doc/source/index.rst [hunk 4]
```diff
@@ -6,10 +6,23 @@
...
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
+and it can generate sentences, evaluation models, inference words. 
+Get started with :ref:`my-reference-label`.
+
+
+Language Model Playground depends on several toolkits, and the document is generated by Sphinx
+
+    * `Pytorch <https://pytorch.org/docs/stable/index.html>`_
```
Comment (ProFatXuanAll) [inline doc/source/index.rst:25]: Use `-` to indicate bullet list.
No need for indentation.
Add description to those dependencies.
Only list import dependencies (choose wisely).
Personally I don't thinks sklearn is that important in this project.

File: doc/source/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,127 @@
+.. _my-reference-label:
+
+QuickStart
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: add space between words

File: doc/source/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+=================
+
+Setup environment
+----------------------
+1. Python version: 3.6+
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: Change to 3.8

Comment (ProFatXuanAll) [reply doc/source/quickstart.rst]: Using inline literal to highlight versions

File: doc/source/quickstart.rst [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+2. CUDA version: 10.0+
+
+Installation and Geting Started
+--------------------------------------
+1. Since the project contains big dataset, so we use git lfs to control versoin. Please insatll git lfs first,
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: ~~so~~ we use \`\`git lfs\`\`
~~insatll~~ install

File: doc/source/quickstart.rst [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+1. Python version: 3.6+
+
+2. CUDA version: 10.0+
+
+Installation and Geting Started
+--------------------------------------
+1. Since the project contains big dataset, so we use git lfs to control versoin. Please insatll git lfs first, 
+and execute
+
+.. code-block:: guess
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: Use shell as language option even if there is no special highlight.
No need to put dollar sign.

File: doc/source/quickstart.rst [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+    $ git lfs install
+
+2. Clone the project from github
+
+.. code-block:: guess
+
+
+    $ git clone https://github.com/ProFatXuanAll/language-model-playground.git
+
+3. Move into the file
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: We use the term *change current directory* more often.

File: doc/source/quickstart.rst [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+2. CUDA version: 10.0+
+
+Installation and Geting Started
+--------------------------------------
+1. Since the project contains big dataset, so we use git lfs to control versoin. Please insatll git lfs first, 
+and execute
+
+.. code-block:: guess
+
+
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst:16]: Redundant empty line.

File: doc/source/quickstart.rst [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+    $ pipenv install
+
+5. Start the virtual Environment
+
+.. code-block:: guess
+
+
+    $ pipenv shell
+
+6. compile the document
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst:43]: This only work if you do `pipenv install --dev` first.

File: doc/source/quickstart.rst [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+
+2. Execute the test
+
+.. code-block:: guess
+
+
+    $ isort .
+    $ autopep8 -r -i -a -a -a lmp
+    $ autopep8 -r -i -a -a -a test
+    $ pipenv run test-coverage
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: We have to run `pipenv run test` first to generate test coverage report later.

File: doc/source/quickstart.rst [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+
+Development Document
+------------------------
+
+1. Make sure your code conform Google python style guide.
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: Now we use numpydoc instead.
Add reference to numpydoc.
See `how_to_doc.rst`.

File: doc/source/quickstart.rst [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+
+2. Do type annotation for every function and method (You might need to see typing).
+
+3. Write docstring for every class, function and method.
+
+4. Run *$ pylint your_code.py* to automatically check your code whether conform to PEP 8.
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: Use inline literal.
Add reference to PEP8.

File: doc/source/quickstart.rst [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+    $ autopep8 -r -i -a -a -a lmp
+    $ autopep8 -r -i -a -a -a test
+    $ pipenv run test-coverage
+
+Development Document
+------------------------
+
+1. Make sure your code conform Google python style guide.
+
+2. Do type annotation for every function and method (You might need to see typing).
```
Comment (ProFatXuanAll) [inline doc/source/quickstart.rst]: Add reference to typing.

**Review by ProFatXuanAll — APPROVED** [review]
